pkg/runctl/controller.go

```diff
@@ -769,13 +812,30 @@ func (c *Controller) addToWorkqueueFromAssociated(obj interface{}) {
 			utilruntime.HandleError(fmt.Errorf("error decoding object tombstone, invalid type"))
 			return
 		}
-		klog.V(3).Infof("Recovered deleted object '%s' from tombstone", object.GetName())
+		klog.V(3).InfoS("Recovered deleted object from unknown state",
+			"recoveredObject", object.GetNamespace()+"/"+object.GetName())
 	}
-	klog.V(4).Infof("Processing object: %s", object.GetSelfLink())
+	klog.V(4).InfoS("Processing the object", "object", object.GetNamespace()+"/"+object.GetName())
 
 	runKey := runmgr.GetPipelineRunKeyAnnotation(object)
 	if runKey != "" {
-		klog.V(4).Infof("Add to workqueue '%s'", runKey)
+		klog.V(4).InfoS("Add to workqueue", "runKey", runKey)
```

Comment:
Use the same key as in line 794

Reply:
Done 503edff454031132e60f244e6d7269674247a5c0